code-goodbuy/goodbuy-nodejs
# Enhancement/get product

With this PR the backend will be enhanced the ability to trigger RabbitMQ which triggers a scraper to get information about products we currently don't have in our database.

**Test Instructions**
- check out to branch enhancement/get_product
- cd into the project folder and run `npm i`
- in file docker-compose.yml change image: goodbuy/nodejs-webserver to image: goodbuy/nodejs-webserver:anthony
- run `npm run docker_dev`
- open the server and pass a product code (which we don't have in our database, just pick something from here: https://www.codecheck.info/essen/suesswaren.kat) and pass it to the URL. Schema: `localhost/product/<code>`

**Expected Behaviour**
- view is not changing
- server responding: 204, `Sending Barcode  <code>` and  `[x] Sent <code>`
- If you refresh the page after a few seconds, the product should appear on the show all product view. Alternatively, you can also look directly in the mongodb
